### PR TITLE
Properly handle standalone component

### PR DESCRIPTION
**Pull request by trasher**
fixes https://github.com/glpi-project/glpi/issues/11864

### CODE REVIEW

File: lib/php/Converter.php [hunk 1]
```diff
@@ -1458,7 +1458,7 @@ public function convertNetworkInventory(array $data): array
 
                     break;
                 case 'components':
-                    $data['content']['network_components'] = is_array($device['components']['component']) ?
+                    $data['content']['network_components'] = isset($device['components']['component'][0]) ?
```
Comment (cedric-anne) [inline lib/php/Converter.php:1461]: If you want to check if array contains only numeric indexes, you can use `is_array($device['components']['component']) && array_is_list($device['components']['component'])`.
It is available through the PHP 8.1 polyfill.

**Review by orthagh — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     },
     "require": {
         "seld/jsonlint": "^1.7",
-        "swaggest/json-schema": "^0.12.31"
+        "swaggest/json-schema": "^0.12.31",
+        "symfony/polyfill-php81": "^1.26"
```
Comment (cedric-anne) [inline composer.json:20]: Polyfill is already included in GLPI 10.0, not sure we have to import it.

Comment (cedric-anne) [reply composer.json:20]: My bad, I thought I was reviewing a plugin PR.

Comment (trasher) [reply composer.json:20]: :)

**Pull request merged**